libs/datasets/sources/cdc_new_vaccine_counties_dataset.py

```diff
@@ -0,0 +1,49 @@
+from libs.datasets import data_source
+from covidactnow.datapublic.common_fields import CommonFields
+from libs.datasets.sources import can_scraper_helpers as ccd_helpers
+
+
+class CDCNewVaccinesCountiesDataset(data_source.CanScraperBase):
+    """In ~June 2021, CDC added a new source for county vaccine data including
+    1st dose data and historical timeseries data. This is surfaced by the
+    scrapers as provider=cdc2 and we now prefer it over the old data source."""
+
+    SOURCE_TYPE = "CDCNewVaccinesCountiesDataset"
+
+    VARIABLES = [
+        ccd_helpers.ScraperVariable(
+            variable_name="total_vaccine_allocated",
+            measurement="cumulative",
+            unit="doses",
+            provider="cdc2",
+            common_field=CommonFields.VACCINES_ALLOCATED,
+        ),
+        ccd_helpers.ScraperVariable(
+            variable_name="total_vaccine_distributed",
+            measurement="cumulative",
+            unit="doses",
+            provider="cdc2",
```

Comment:
Will we be able to remove cdc2 at some point?

Reply:
or rather rename cdc2 -> cdc

Reply:
cc/ @smcclure17 Not 100% sure what this looks like on the scraper side, but I think we probably could if we want to...  In theory we could make the scraper publish to both `cdc` and `cdc2` for a little while, then migrate this, and then delete the `cdc2` data.

One of the tricky things I'm not sure about is the other vaccine variables (distributed, allocated) that aren't available through `cdc2` right now...

Reply:
I think we could. The vaccine distributed data comes from the CDC state-level scraper (which is still active), not the old county-vaccine scraper. Also, the new CDC county source has all the same variables as the old CDC county source, but I didn't add them as I didn't think we used/wanted them.